### ISSUE TITLE
Local subscribtion storage

### ISSUE THREAD

**Issue by Stenledtrap** [krawieck/lemmur#223]
### Describe the solution you'd like

Being able to store subscribtions on local storage without requiring a log in.
Also import/export options for the subacription file.

**Comment by Stenledtrap** [comment]
> 
> ### Describe the solution you'd like
> A clear and concise description of what you want to happen.
> 
> Being able to store subscribtions on local storage without requiring a log in.
> Also import/export options for the subscription file.

> 
>

**Comment by krawieck** [comment]
i don't think we will ever do that. but i think we might mitigate a need for that if we add custom home list (sth like multireddits). i have trouble coming up with a name for this feature. maybe we could call it multilemmy
